bahmni-emr-api/src/main/java/org/openmrs/module/bahmniemrapi/laborder/mapper/LabOrderResultMapper.java

```diff
@@ -88,9 +92,28 @@ private Obs newResultObs(Order testOrder, Date obsDate, Concept concept, LabOrde
             return null;
         }
         obs.setValueAsString(labOrderResult.getResult());
+        checkResultRangesForAbsolutes(obs, accessionUuid);
         return obs;
     }
 
+    /**
+     * This method just logs error if the results are out of absolute ranges. This will be errored out (ValidationException)
+     * by openmrs by the {@link org.openmrs.validator.ObsValidator} during save
+     */
+    private void checkResultRangesForAbsolutes(Obs obs, String accessionUuid) {
+        if (obs.getConcept().isNumeric()) {
+            if (obs.getValueNumeric() != null) {
+                ConceptNumeric cn = (ConceptNumeric) obs.getConcept();
```

Comment:
Can we consider using this method of OpenMRSUtil?
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsUtil.java#L195

Method: `isInAbsoluteNumericRange ()`

Reply:
I knew about this util method. I wanted to print which range it is beyond to provide more info. Also the method returns false if any of the Absolute High or Low is not defined. However, the validation will still fail if one of them is mentioned, which the util method does not cover. For example, in CIEL, we have many numeric test concepts where a concept does not have the "absolute high" is not defined but "absolute low" is defined. (136 as of now). and visa versa.